### PR TITLE
Add rule unhook-a-dll.yml

### PR DESCRIPTION
**Pull request by jtothej**
Add rule unhook-a-dll.yml

### CODE REVIEW

File: anti-analysis/anti-av/unhook-a-dll.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+rule:
+  meta:
+    name: unhook a DLL
```
[the remark this answers is not in the document]
Comment (jtothej) [reply anti-analysis/anti-av/unhook-a-dll.yml:3]: I initially created the rule specifically for NTDLL but then decided to make it more generic. Most likely NTDLL will be the unhooking target in most samples anyway but I wanted to match also on less common DLLs. If you think the rule is too broad I'll change it to its initial form.

File: anti-analysis/anti-av/unhook-a-dll.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+      - Defense Evasion::Impair Defenses::Disable or Modify Tools [T1562.001]
+    mbc:
+      - Defense Evasion::Disable or Evade Security Tools [F0004]
+    references:
+      - https://www.ired.team/offensive-security/defense-evasion/how-to-unhook-a-dll-using-c++
+    examples:
+      - 282c32269a9893c5741ca682268369421c43ac21d73b1f6c23386d61c93bf3e9:0x1400014A4
+  features:
+    - and:
+      - api: GetModuleHandle
```
Comment (mr-tz) [inline anti-analysis/anti-av/unhook-a-dll.yml:20]: add `GetModuleHandleEx`?

File: anti-analysis/anti-av/unhook-a-dll.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (mr-tz) [reply anti-analysis/anti-av/unhook-a-dll.yml:3]: hm, I think it shouldn't result in many FPs and if so, we can fine tune it
for when we do come back to it later, can you please add this comment to the rule?

[the remark this answers is not in the document]
Comment (mr-tz) [reply anti-analysis/anti-av/unhook-a-dll.yml:3]: and how about renaming to better reflect what's captured here, e.g. `overwrite DLL .text section to remove hooks` or similar?

File: anti-analysis/anti-av/unhook-a-dll.yml (hunk 2 above)
Comment (jtothej) [reply anti-analysis/anti-av/unhook-a-dll.yml]: Added in https://github.com/mandiant/capa-rules/pull/921/commits/3a03215dd576ad790256398d8b4999bffcb16e1d

File: anti-analysis/anti-av/unhook-a-dll.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (jtothej) [reply anti-analysis/anti-av/unhook-a-dll.yml]: Thanks for suggestions! Updated the rule in https://github.com/mandiant/capa-rules/pull/921/commits/3a03215dd576ad790256398d8b4999bffcb16e1d

**Pull request merged**